docs/data/data-grid/rows/rows.md

```diff
@@ -146,14 +146,26 @@ Row spanning allows to change this default behavior.
 It allows cells to span multiple rows.
 This is very close to the "row spanning" in an HTML `<table>`.
 
-## 🚧 Row reorder [<span class="plan-pro"></span>](https://mui.com/store/items/material-ui-pro/)
-
-> ⚠️ This feature isn't implemented yet. It's coming.
->
-> 👍 Upvote [issue #206](https://github.com/mui/mui-x/issues/206) if you want to see it land faster.
+## Row reorder [<span class="plan-pro"></span>](https://mui.com/store/items/material-ui-pro/)
 
 Row reorder is used to rearrange rows by dragging the row with the mouse.
 
+By default, `DataGridPro` does not allow row reordering. To enable reordering on all rows by dragging the special row reordering cells and moving it up or down, use the `rowReordering` prop.
+
+{{"demo": "RowOrderingGrid.js", "disableAd": true, "bg": "inline"}}
+
+To capture changes in the order of the dragged row use the `onRowOrderChange` callback.
+
+In addition, row reordering emits the following events that can be imported:
+
+- `rowDragStart`: emitted when dragging of a row starts.
+- `rowDragOver`: emitted when dragging a row over another row.
+- `rowDragEnd`: emitted when dragging of a row stops.
+
+> ⚠️ For now, row reordering is disabled if working is applied to the grid.
+>
+> In addition, if row grouping or Tree Data is used the row reordering is also disabled.
```

Comment:
Maybe move it to the warning above so that all cases when reordering is disabled are in one place?

Reply:
```suggestion
> In addition, if row grouping or tree data is being used, the row reordering is also disabled.
```

Is "A is used" a valid formulation ?